migrate_pim/upgrade_minor_version.rst

```diff
@@ -0,0 +1,52 @@
+How to upgrade to a minor version?
+==================================
+
+We release a minor version every 3 months, these versions bring new features.
```

Comment:
We release a minor version 3 times a year: the X.1 in May (STS: Short Term Support version), the X.2 in August (also a STS - Short Term Support - version) and the X.3 in November (LTS: Long Term Support version). These versions bring new features.

Reply:
done